person.rb

```diff
@@ -1,12 +1,26 @@
-class Person
+require_relative './nameable'
+require_relative './capitalizedecorator'
+require_relative './trimmer_decorator'
+
+class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- [ ] For rentals to be both accessed and edited by a person, it needs to be given both getters and setters methods like name and age, kindly add ```rentals``` to ```attr_accessor```

Reply:
@Ghiftee  Thank you for review